simtools/model/camera.py

```diff
@@ -223,6 +223,17 @@ def getPixelDiameter(self):
         """
         return self._pixels["pixel_diameter"]
 
+    def getPixelActiveSolidAngle(self):
+        """
+        Get the active solid angle of a pixel in sr.
+        """
+
+        pixelArea = self.getPixelDiameter() ** 2
+        # In case we have hexagonal pixels:
+        if self.getPixelShape() == 1 or self.getPixelShape() == 3:
```

Comment:
Not related to this function, but I think it is not a good solution to encode in the pixel shape the pixel rotation angle (I was initially confused that we have two different pixel shapes for hexagonal pixels)

Nothing to fix, just a remark.

Reply:
You are right, but as "usual" this comes directly from sim_telarray. Not much we could do about it at the moment.